plugins/services/src/js/constants/OperatorTypes.js

```diff
@@ -1,38 +1,55 @@
+const DEFAULT_HELP = "A string, integer or regex value.";
+const OPTIONAL_HELP = `${DEFAULT_HELP} This field is optional.`;
+
 const OperatorTypes = {
   UNIQUE: {
     requiresValue: false,
     requiresEmptyValue: true,
-    stringNumberValue: false
+    stringNumberValue: false,
+    tooltipContent: "The unique operator does not accept a value.",
```

Comment:
same thing here, is it possible to calculate this based on `requiresEmptyValue`?